src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1736,12 +1736,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
          that can't multiplatform.  -->
     <ItemGroup Condition="'@(_ProjectsWithPlatformAssignment)' != ''">
       <ProjectsWithNearestPlatform Include="@(_ProjectsWithPlatformAssignment)"/>
-      <ProjectsWithNearestPlatform Condition="'@(ProjectsWithNearestPlatform)' == '%(Identity)' and '%(ProjectsWithNearestPlatform.NearestPlatform)' != ''">
+      <ProjectsWithNearestPlatform Condition="'%(ProjectsWithNearestPlatform.Identity)' != '' and '%(ProjectsWithNearestPlatform.NearestPlatform)' != ''">
```

Comment:
For this one and others that have only a single metadatum in the rest of the condition, batching over `%(Identity)` is not required. However the batching behavior in the "`NearestTargetFramework` is set and `HasSingleTargetFramework` is not `true`" conditions is pretty baffling so it's needed there and I'm keeping it in all of these cases for consistency.